frontend/components/commons/results/ResultsList.vue

```diff
@@ -16,32 +16,34 @@
   -->
 
 <template>
-  <div class="list">
+  <div class="content">  
     <slot name="header" />
     <div class="results-scroll" id="scroll">
       <div :style="{ paddingTop: `${dataset.viewSettings.headerHeight + 10}px` }" v-if="showLoader">
         <results-loading :size="dataset.viewSettings.pagination.size" />
       </div>
       <DynamicScroller
-        v-else
         page-mode
         class="scroller"
         :items="visibleRecords"
         :min-item-size="150"
         :buffer="200"
         :style="{ paddingTop: `${dataset.viewSettings.headerHeight + 10}px` }"
       > 
+        <template #before>
+          <slot name="pre-records-area"/>
```

Comment:
Could we use the `results-header` name?

Reply:
This "header" will be shown if no results was found?

Reply:
If no results, this header is hidden